### DISCUSSION TITLE
Welcome to Data-Science-For-Beginners Discussions!

### DISCUSSION THREAD

**Discussion by jlooper** [microsoft/Data-Science-For-Beginners#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anupamishra333** [comment]
@jlooper I have reported a bug that the link of Notebook is not working on Github pages. already Created issue. Kindly Review this

**Reply by jlooper** [reply]
got it, thanks, and addressed

**Comment by femmetech** [comment]
Hello everyone! I am glad I came across this opportunity as a beginner to connect and learn. This is a new departure into data science fundamentals but part of a bigger journey to eventually transition into this industry. Would love to connect with others starting out / beginning (especially as an older 50 yr old😊) in this fomain.

**Reply by jlooper** [reply]
welcome @femmetech - I'm 50 too! Super happy to see diverse learners come to our repos and we hope you enjoy the learning experience. I made a [TikTok](https://www.tiktok.com/@javascriptauntie/video/6912659037674851590?is_copy_url=0&is_from_webapp=v1&sender_device=pc&sender_web_id=7010784245686650374) just for us ☺️

**Comment by riddhimaaa** [comment]
I'd like to know if this is for learning purposes only or is there any kind of certification available for this?

**Reply by jlooper** [reply]
It’s for learning purposes only 🙏

**Reply by riddhimaaa** [reply]
oh, that is very very helpful anyways. Thank you so much.

**Comment by femmetech** [comment]
Looking forward to connecting. Are there opportunities for accountability
partners, small projects etc ( Think it would be fun & a great handson)

**Comment by CinnamonXI** [comment]
Hello @jlooper I think the video on Defining data science (https://youtu.be/pqqsm5reGvs)
has a wrong content. It is talking about probability.
It is located in this readme(https://github.com/microsoft/Data-Science-For-Beginners/tree/main/1-Introduction/01-defining-data-science)

**Reply by jlooper** [reply]
hi, we fixed this a while ago, so closing this

**Comment by IKpukka** [comment]
Hello
How about we get solutions to the assignments so we can know if we are on track with the learning process

Thanks

**Comment by ccarlson0225** [comment]
Hello everyone my name is Chris and I'm gonna be honest I have no idea what I'm doing and used to joke that I'm computer illiterate. A few weeks ago I got into cryptocurrency and have been obsessed ever since. That has lead me to want to know how to code and how it works. So I'm here to figure it out.

**Comment by jobartucz** [comment]
I'm a little disappointed that there is not more of a humanistic / social justice focus in this curriculum. I was hoping to empower students and show them how using these powerful tools can have an effect on their own lives, their communities, and the larger world.

**Reply by jlooper** [reply]
Hello, there is a lesson on Real World usage, but we also have a lesson that might be of interest to you in our ML curriculum (aka.ms/ml-beginners) where we include a lesson on Ethics as well as a round-up of real-world examples. Let me know what you think!

**Comment by N-CALLISTUS** [comment]
Hi, my name is Callistus.
I'm new here and I'm glad to start this journey.
Please I am confused on how to start.
What are the necessary applications I need to install on my computer to get started with this journey?

**Reply by ioguntayo** [reply]
You can install Anaconda Navigator, it contains the IDE environments and most of the tools you will need

**Comment by DS-Ozzy** [comment]
Hi everyone.
I'm trying to run the notebook.ipynb code but I can't find any button or anything else to do that. I've already clicked everything but nothing happen.

**Reply by amochuko** [reply]
<img width="945" alt="Screenshot 2022-12-07 at 12 13 45" src="https://user-images.githubusercontent.com/7204868/206165009-abef3015-65ff-4882-8321-0cec2676c4c5.png">

If you installed the Jupyter Notebook plugin/addOn correctly on your editor, you should see the above screenshot (VS Code) helpful.

**Comment by ioguntayo** [comment]
My name is Ibitayo Oguntayo. I have an excellent understanding of RDBMS. I have foundation knowledge of ML (Udacity). I believe this should be natural for me.

**Comment by Kleint27** [comment]
Yes

**Comment by treasurekunde** [comment]
Hello everyone. 
My name is Treasure Kunde and I'm excited to be a part of this learning process and community.
I just started the lessons a while ago and being able to understand this on a step-by-step basis is fun although, I really can't tell my progress in the assignments. I'll be happy to get some help on that! Thank you.
